aten/src/ATen/cpu/vec/vec256/vec256_half_neon.h

```diff
@@ -565,9 +565,9 @@ class Vectorized<c10::Half> {
   }
 
   Vectorized<c10::Half> operator!=(const Vectorized<c10::Half>& other) const {
-    float32x4_t r0 = vreinterpretq_f16_u16(
+    float16x8_t r0 = vreinterpretq_f16_u16(
```

Comment:
🥇 
Do you mind submitting it as a separate PR?

Also, I wonder how it passed the testing/compiled? Does compile inserts some sort of implicit conversion here?
cc: @metascroy , @jgong5

Reply:
Ok, I think I know what it refers to here, see https://godbolt.org/z/xoasoGM81